### PR TITLE
CCI-1697 feat: update allocation docs to include the single metric

### PR DESCRIPTION
**Pull request by jeannei**
### What does this PR do? What is the motivation?

* Updated container allocation docs with new information about ccm single metric allocation.
* Move old metrics into its own section to discourage use
* Add docs around new allocation tags

#### Preview

* Doc [preview](https://docs-staging.datadoghq.com/jean/update-ccm-allocation-docs/cloud_cost_management/container_cost_allocation/#understanding-spend)

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 1]
```diff
@@ -62,7 +65,7 @@ Based on the CPU or memory usage of each task (as reported in the CUR), Datadog
 
 Once all tasks have been assigned a cost based on their resource reservations, some instance cost is left over. This is the cost of unreserved resources, which is called **Cluster Idle** cost. This cost is assigned the `is_cluster_idle` tag, and it represents the cost of resources not reserved by any ECS tasks. For more information, see the [Understanding cluster idle cost](#understanding-cluster-idle-cost) section.
 
-If you enabled AWS Split Cost Allocation, ECS tasks will have costs allocated based on their actual usage, with an additional `is_workload_idle` tag that will represent any usage that is requested but unused. For instance, a task that requests 2GB of memory but only uses 1GB will have the remaining 1GB worth of cost tagged as `is_workload_idle`. 
+If you enabled AWS Split Cost Allocation, ECS tasks will have costs allocated based on their actual usage, with an additional `is_workload_idle` tag that will represent any usage that is requested but unused. For instance, a task that requests 2GB of memory but only uses 1GB will have the remaining 1GB worth of cost tagged as `is_workload_idle`.
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:68]:
```suggestion
If you enable AWS Split Cost Allocation, ECS tasks allocate costs based on their actual usage, with an additional `is_workload_idle` tag that represent any usage that is requested but unused. For example, a task that requests 2GB of memory but only uses 1GB has the remaining 1GB worth of cost tagged as `is_workload_idle`.
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 2]
```diff
@@ -74,34 +77,28 @@ Any cost other than EC2, computed for instances hosting Kubernetes pods or ECS t
...
-- If you have a mix of CPU and memory-intensive workloads, or you simply want a consistent way to visualize the costs of considering all resources, you can use formulas and functions to create your own equation to combine CPU and memory costs. For example, for a 50/50 split, you can plot:
-
-$$\text"a " = \text" sum(aws.cost.amortized.cpu.allocated) by {team}"$$
-
-$$\text"b" = \text"sum(aws.cost.amortized.mem.allocated) by {team}"$$
-
-$$\text"mixed_cost" = (0.5 * \text"a") + (0.5 * \text"b")$$
-
-As long as the multipliers are positive and sum to 1, the formula will generate coherent costs.
+- If you have a mix of CPU and memory-intensive workloads, or you simply want a consistent way to visualize the costs of considering all resources, using `.shared.resources.allocated` will meet your use case.
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:89]:
```suggestion
- If you have a mix of CPU and memory-intensive workloads, or you want a consistent way to visualize the costs of considering all resources, use `.shared.resources.allocated`.
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 3]
```diff
@@ -74,34 +77,28 @@ Any cost other than EC2, computed for instances hosting Kubernetes pods or ECS t
...
   - `aws.cost.amortized.cpu.allocated`
   - `aws.cost.amortized.mem.allocated`
+  - `aws.cost.amortized.shared.resources.allocated`
 
 ## Understanding cluster idle cost
 
 Cluster idle cost is the cost of CPU or memory resources that are not reserved by any workloads.
 
-You can visualize this cost at the Kubernetes node, ECS host, or cluster level, using the `is_cluster_idle` tag. All costs allocated to scheduled pods or tasks have `is_cluster_idle: N/A`, while all compute costs not assigned to any pods or tasks have the tag `is_cluster_idle: true`.
+You can visualize this cost at the Kubernetes node, ECS host, or cluster level, using the `is_cluster_idle` tag. All costs allocated to scheduled pods or tasks have `is_cluster_idle: N/A`, while all compute costs not assigned to any pods or tasks have the tag `is_cluster_idle: true`. *Note if using `.shared.resources.allocated` metrics, use the `allocated_spend_type` tag.*
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:101]:
```suggestion
You can visualize this cost at the Kubernetes node, ECS host, or cluster level, using the `is_cluster_idle` tag. All costs allocated to scheduled pods or tasks have `is_cluster_idle: N/A`, while all compute costs not assigned to any pods or tasks have the tag `is_cluster_idle: true`. **Note**: if using `.shared.resources.allocated` metrics, use the `allocated_spend_type` tag.
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 4]
```diff
@@ -113,15 +110,23 @@ Here is a sample query that displays all Kubernetes cluster cost, broken down by
 
 Datadog consolidates and applies additional tags from various sources to cost metrics. Conflicts are resolved by favoring higher-specificity tags such as pod tags over lower-specificity tags such as host tags. For example, if a Kubernetes pod has the tag `service:datadog-agent` and the node it runs on has the tag `service:aws-node`, the resulting final tag shows up as `service:datadog-agent`.
 
+### Containers
+
+| Out-of-the-box tag  |  Description |
+| ---                 | ------------ |
+| `orchestrator`      | The orchestration platform associated with the item, e.g. kubernetes, ecs. |
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:117]:
```suggestion
| `orchestrator`      | The orchestration platform associated with the item (kubernetes, ecs) |
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 5]
```diff
@@ -113,15 +110,23 @@ Here is a sample query that displays all Kubernetes cluster cost, broken down by
...
+| `allocated_spend_type`     | The spend category category assocuated with the item, e.g. usage, cluster_idle, workload_idle. *Only available for `.shared.resources.allocated` metrics.* |
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:118]:
```suggestion
| `allocated_spend_type`     | The spend category category assocuated with the item (usage, cluster_idle, workload_idle). *Only available for `.shared.resources.allocated` metrics.* |
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 6]
```diff
@@ -113,15 +110,23 @@ Here is a sample query that displays all Kubernetes cluster cost, broken down by
...
+| `allocated_resource`   | The resource category associated with the item, e.g. cpu, memory. *Only available for `.shared.resources.allocated` metrics.* |
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:119]:
```suggestion
| `allocated_resource`   | The resource category associated with the item (cpu, memory). *Only available for `.shared.resources.allocated` metrics.* |
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 7]
```diff
@@ -35,12 +35,15 @@ When the prerequisites are met, new AWS cost metrics automatically appear.
 | `aws.cost.net.amortized.mem.allocated` | Net EC2 costs allocated by memory requested by a pod or ECS task <br> *Based on `aws.cost.net.amortized`, if available* |
 | `aws.cost.amortized.cpu.allocated` | EC2 costs allocated by CPU requested by a pod or ECS task <br> *Based on `aws.cost.amortized`* |
 | `aws.cost.net.amortized.cpu.allocated` | Net EC2 costs allocated by CPU requested by a pod or ECS task <br> *Based on `aws.cost.net.amortized`, if available* |
+| `aws.cost.amortized.shared.resources.allocated` | EC2 costs allocated by CPU & memory requested by a pod or ECS task, using a 60:40 split for CPU & memory respectively. <br> *Based on `aws.cost.amortized`* |
+| `aws.cost.net.amortized.shared.resources.allocated` | Net EC2 costs allocated by CPU requested by a pod or ECS task, using a 60:40 split for CPU & memory respectively. <br> *Based on `aws.cost.net.amortized`, if available* |
```
Comment (enemyghost) [inline content/en/cloud_cost_management/container_cost_allocation.md:39]: Should we show an image of the analytics controls, displaying how to select a container cost metric? The metric names don't appear anywhere on the page.

**Review by enemyghost — COMMENTED** [review]
Consider moving info about the legacy metrics into their own section, or removing them entirely. 

Right now, parsing all the information about how all the metrics work differently is a bit confusing. Since we want to push people to use the new one (and will eventually deprecate the old ones), I'd prefer the docs to be very focused on how the new one works and what it means.

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 8]
```diff
@@ -31,17 +31,14 @@ When the prerequisites are met, new AWS cost metrics automatically appear.
...
-| `aws.cost.net.amortized.cpu.allocated` | Net EC2 costs allocated by CPU requested by a pod or ECS task <br> *Based on `aws.cost.net.amortized`, if available* |
+| `aws.cost.amortized.shared.resources.allocated` | EC2 costs allocated by CPU & memory requested by a pod or ECS task, using a 60:40 split for CPU & memory respectively. <br> *Based on `aws.cost.amortized`* |
+| `aws.cost.net.amortized.shared.resources.allocated` | Net EC2 costs allocated by CPU & memory requested by a pod or ECS task, using a 60:40 split for CPU & memory respectively. <br> *Based on `aws.cost.net.amortized`, if available* |
+
 
 These new cost metrics include all of your AWS cloud costs. This allows you to continue visualizing all of your cloud costs at one time, with added visibility into pods and tasks running on EC2 instances.
 
 For example, say you have the tag `team` on S3 buckets, RDS stores, and Kubernetes pods. You can use one of the new metrics to group cost by `team`, and each group then includes the S3 and RDS costs for that team, as well as the cost of compute resources reserved by the tagged pods.
```
Comment (ktaylor19) [inline content/en/cloud_cost_management/container_cost_allocation.md:40]: not just reserved its max (reserved, used)

Comment (jeannei) [reply content/en/cloud_cost_management/container_cost_allocation.md:40]: We are just going to update text to say `used`. Trying to explain using the maximum of reserved and used made the section too wordy.

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 9]
```diff
@@ -132,8 +106,23 @@ In addition to ECS task tags, the following out-of-the-box tags are applied to c
 | `ecs_cluster_name`      | The name of the ECS cluster. |
 | `is_aws_ecs`            | All costs associated with running ECS. |
 | `is_aws_ecs_on_ec2`     | All EC2 compute costs associated with running ECS on EC2. |
-| `is_cluster_idle`       | The cost of unreserved CPU or memory on EC2 instances running ECS tasks. |
 | `is_aws_ecs_on_fargate` | All costs associated with running ECS on Fargate. |
+| `is_cluster_idle`       | The cost of unreserved CPU or memory on EC2 instances running ECS tasks. *Only available for `.cpu.allocated` or `.mem.allocated` metrics.*|
+
+## Other cost metrics
+
+In addition to the `.shared.resources.allocated` metrics, Datadog also allows you to visulaize your workload costs by CPU or memory only.
```
Comment (estherk15) [inline content/en/cloud_cost_management/container_cost_allocation.md:114]:
```suggestion
In addition to the `.shared.resources.allocated` metrics, Datadog also allows you to visualize your workload costs by CPU or memory only.
```

**Review by estherk15 — APPROVED** [review]

**Review by enemyghost — APPROVED** [review]
